spec/word_spec.rb

```diff
@@ -0,0 +1,45 @@
+require_relative '../word_parser.rb'
+
+describe 'Word parser' do
+  
+  let(:parser) { WordParser }
```

Comment:
what does this `let` statement get you over just repeating the name of the module?